dev/MRTCore/mrt/Core/unittests/MrmTests.cpp

```diff
@@ -563,35 +590,37 @@ TEST_CLASS(BasicTest)
     TEST_METHOD(GetFilePath)
     {
         wchar_t* path;
-        Assert::AreEqual(MrmGetFilePathFromName(L"resources.pri", &path), S_OK);
-        Assert::IsNotNull(wcsstr(path, L"resources.pri"));
+        VERIFY_ARE_EQUAL(MrmGetFilePathFromName(L"resources.pri", &path), S_OK);
+        VERIFY_IS_NOT_NULL(wcsstr(path, L"resources.pri"));
         MrmFreeResource(path);
 
-        Assert::AreEqual(MrmGetFilePathFromName(L"something.pri", &path), S_OK);
+        VERIFY_ARE_EQUAL(MrmGetFilePathFromName(L"something.pri", &path), S_OK);
         // Even if the file doesn't exist, we will still return a path. This is for those don't use PRI for resource.
-        Assert::IsNotNull(wcsstr(path, L"something.pri"));
+        VERIFY_IS_NOT_NULL(wcsstr(path, L"something.pri"));
         MrmFreeResource(path);
 
-        Assert::AreEqual(MrmGetFilePathFromName(nullptr, &path), S_OK);
-        Assert::IsNotNull(wcsstr(path, L"resources.pri"));
+        VERIFY_ARE_EQUAL(MrmGetFilePathFromName(nullptr, &path), S_OK);
+        VERIFY_IS_NOT_NULL(wcsstr(path, L"resources.pri"));
         MrmFreeResource(path);
     }
 
 private:
     void VerifyQualifierValue(UINT32 qualifierCount, PWSTR* qualifierNames, PWSTR* qualifierValues, PCWSTR name, PCWSTR expectedValue)
     {
-        Assert::IsTrue(qualifierCount > 0);
+        VERIFY_IS_TRUE(qualifierCount > 0);
         bool found = false;
         for (UINT32 i = 0; i < qualifierCount; i++)
         {
             if (0 == _wcsicmp(name, qualifierNames[i]))
             {
-                Assert::IsTrue(0 == _wcsicmp(expectedValue, qualifierValues[i]));
+                VERIFY_IS_TRUE(0 == _wcsicmp(expectedValue, qualifierValues[i]));
```

Comment:
VERIFY_ARE_EQUAL

Reply:
I'll replace this with the string helper.